### PR TITLE
Email Python

### PR DESCRIPTION
**Pull request by JLodha**
__21__

#### Short description of what this resolves:
Added the python file to send message through command line

#### Changes proposed in this pull request and/or Screenshots of changes:

-
-
-

### CODE REVIEW

**Comment by Lokesh2703** [comment]
See the updated instructions now and proceed.

**Comment by Lokesh2703** [comment]
See the updated instructions in the issue section now and proceed.

**Comment by JLodha** [comment]
Please have a look now sir

File: Submissions/JLodha/Task_3/mail.py [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
-connection.login("iit2019134@iiita.ac.in", "PASSWORDHERE")
-message = "IIITian"
-connection.sendmail("iit2019134@iiita.ac.in", "iit2019132@iiita.ac.in", message)  
+id = getpass.getpass('id:')
+pwd = getpass.getpass('password: ')
+connection.login(id, pwd)
+rec = input("Enter the person whom you want to send the mail: ")
+subject = input("Subject: ")
+message = input('Message to Send: ')
+finalmess = 'Subject: ' + subject+  "\n" + "Message: " + message
```
Comment (Lokesh2703) [inline Submissions/JLodha/Task_3/mail.py:10]: There must be one more line to get content in  the mail.

File: Submissions/JLodha/Task_3/mail.py [hunk 2]
```diff
@@ -1,7 +1,12 @@
-import smtplib 
+import smtplib,getpass
 connection = smtplib.SMTP('smtp.gmail.com', 587)  
 connection.starttls()
-connection.login("iit2019134@iiita.ac.in", "PASSWORDHERE")
-message = "IIITian"
-connection.sendmail("iit2019134@iiita.ac.in", "iit2019132@iiita.ac.in", message)  
+id = getpass.getpass('id:')
```
Comment (Lokesh2703) [inline Submissions/JLodha/Task_3/mail.py:4]: Let the user see their username.

**Pull request merged**